core/src/prim/text.rs

```diff
@@ -226,6 +226,11 @@ impl TextOp {
       (Self::LenChars, Text(xs)) => Some(Nat(xs.len_chars().into())),
       (Self::LenBytes, Text(xs)) => Some(Nat(xs.len_bytes().into())),
       (Self::LenLines, Text(xs)) => Some(Nat(xs.len_lines().into())),
+      (Self::ToBytes, Text(xs)) => if xs.len_bytes() == 0 {
```

Comment:
Shouldn't we be able to still make the conversion if `xs.len_bytes() == 0` to `Bytes(vec![])`?

Reply:
I've removed the check, so it should be able to return an empty vector now.